Création de kamehameha dans Produit ! (#4159)
betagouv/beta.gouv.fr: content/_startups/kamehameha.md

---
title: "Kamehameha "
mission: Lutter contre le harcèlement des jeunes
owner: "Agglomération de Pau "
incubator: anct
contact: incubateur@anct.gouv.f
events: []
phases:
  - name: investigation
    start: 2019-11-01
    end: 2020-01-31
  - name: alumni
    start: 2020-02-01
---
**Contexte**

> En France, près de 700 000 enfants sont victimes de harcèlement scolaire chaque année, soit environ 3 élèves par classe. A Pau, comme dans d’autres villes, les jeunes interrogés décrivent des situations de violence entre pairs au sein de leurs établissements scolaires.

Reviewer: ```suggestion
En France, près de 700 000 enfants sont victimes de harcèlement scolaire chaque année, soit environ 3 élèves par classe. À Pau, comme dans d’autres villes, les jeunes interrogés décrivent des situations de violence entre pairs au sein de leurs établissements scolaires.
```

* Le traitement du problème de harcèlement entre élèves varie d’un établissement à l’autre.
* La définition du harcèlement scolaire, son caractère répétitif et grave, est maîtrisée mais pas intégrée. Certains adultes parlent encore de problèmes liés aux préjugés et excluent facilement l’hypothèse de harcèlement scolaire en parlant de conflit ponctuel. Les élèves, quant à eux, décrivent des situations de violence répétitives sans pour autant les qualifier de harcèlement. On en déduit que les violences physiques et verbales au collège sont vécues comme étant normales, voire inévitables.
* Les lieux de harcèlement scolaire sont pluriels : tous les lieux de sociabilisation et souvent de transition : le bus, le couloir, la cour de l’école, les toilettes, la cantine, etc... Le téléphone portable et les réseaux sociaux peuvent être considérés comme des lieux de transition. Ils sont à la fois utilisés dans les établissements scolaires et en dehors
* Les crispations du personnel encadrant les jeunes sur ce sujet, notamment en raison du manque de temps pour gérer cette responsabilité et du caractère délicieux du harcèlement.

> * L’existence de nombreuses solutions de prévention et de traitement du harcèlement entre élèvesIl existe des solutions préventives mises en place pour faire travailler sur l’empathie aux jeunes : le théâtre forum, la réalisation de spots, ou films préventifs, de jeux, de serious game, etc. Dans le cadre de NonAuHarcèlement, un protocole de traitement du harcèlement scolaire préconise notamment des entretiens avec le.s témoin.s. Une solution de traitement de la violence appelée méthode Palo Alto est proposée aux jeunes de Pau. Il s’agit d’une thérapie brève résolutoire qui aide l’individu harcelé à changer de posture. Il n’y a pas ici recours à la figure du témoin ou de l’adulte.

Reviewer: ```suggestion
* L’existence de nombreuses solutions de prévention et de traitement du harcèlement entre élèves.Il existe des solutions préventives mises en place pour faire travailler sur l’empathie aux jeunes : le théâtre forum, la réalisation de spots, ou films préventifs, de jeux, de serious games, etc. Dans le cadre de NonAuHarcèlement, un protocole de traitement du harcèlement scolaire préconise notamment des entretiens avec le.s témoin.s. Une solution de traitement de la violence appelée méthode Palo Alto est proposée aux jeunes de Pau. Il s’agit d’une thérapie brève résolutoire qui aide l’individu harcelé à changer de posture. Il n’y a pas ici recours à la figure du témoin ou de l’adulte.
```

**Hypothèses et intuitions**

* Les réseaux sociaux accentuent et initient de nouveaux phénomènes d’harcèlement\
  Par le biais des nouveaux outils numériques, le harcèlement scolaire se professionnalise. Des jeunes utilisent de plus en plus le détournement d’images ou d’enregistrements sonores qu’ils fabriquent pour nuire à la réputation d’un autre jeune et lancer une rumeur sur les réseaux sociaux (Youtube, Snapchat, Instagram, Tik Tok...)
* La figure du témoin est déjà sollicitée pour lutter contre le harcèlement entre élèves. La place du témoin a été largement exploitée dans tous les dispositifs et les préconisations formulés par le ministère de l’Education nationale et de la Jeunesse. Non Au Harcèlement propose notamment la mise en place d’ambassadeurs.rices dans les établissements.
* L’isolement des enfants comme critère potentiellement objectif d’une situation d’harcèlement. Afin de « resserrer » le problème, l’une des hypothèses envisagées était de proposer une solution pour les enfants, les jeunes qui seraient mis à l’écart du groupe. La piste n’a pas encore été suivie en raison d’un problème d’accès au terrain d’expérimentation.

> * L’intervention d’un adulte peut aggraver la situation de victime de l’enfant. L’intervention d’un l’adulte maintient l’enfant dans une position de victime et de « faible » aux yeux du groupe. Par ailleurs, les jeunes qui vont vers un adulte pour exprimer un problème peuvent subir des représailles.

Reviewer: ```suggestion
* L’intervention d’un adulte peut aggraver la situation de victime de l’enfant. L’intervention d’un adulte maintient l’enfant dans une position de victime et de « faible » aux yeux du groupe. Par ailleurs, les jeunes qui vont vers un adulte pour exprimer un problème peuvent subir des représailles.
```
